data_steward/validation/metrics/completeness.py

```diff
@@ -1,7 +1,13 @@
+# Python imports
+from io import open
+
+# Third party imports
+
+# Project imports
 import bq_utils
 import resources
+from common import CDM_TABLES, PII_TABLES
```

Comment:
I'm not seeing `PII_TABLES` being utilized. Is that import necessary?

Reply:
You're right, I was using this earlier and forgot to remove it.